pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -1061,8 +1061,19 @@ func (m *ManagerImpl) sanitizeNodeAllocatable(node *schedulerframework.NodeInfo)
 func (m *ManagerImpl) isDevicePluginResource(resource string) bool {
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
-	_, registeredResource := m.healthyDevices[resource]
+	var registeredResource bool
+	if utilfeature.DefaultFeatureGate.Enabled(features.DRAExtendedResource) {
```

Comment:
in your testing, will it flip the node status back and forth on kubelet restart when Device plugin takes time to re-register and we picked up devices from the checkpoint file? I wonder if it may cause some strange race conditions?